x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint/response_console.cy.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Agent } from '@kbn/fleet-plugin/common';
+import {
+  inputConsoleCommand,
+  openResponseConsoleFromEndpointList,
+  performCommandInputChecks,
+  submitCommand,
+  waitForCommandToBeExecuted,
+  waitForEndpointListPageToBeLoaded,
+} from '../../tasks/response_console';
+import type { IndexedFleetEndpointPolicyResponse } from '../../../../../common/endpoint/data_loaders/index_fleet_endpoint_policy';
+import {
+  getAgentByHostName,
+  getEndpointIntegrationVersion,
+  reassignAgentPolicy,
+} from '../../tasks/fleet';
+import { checkEndpointListForIsolatedHosts, createAgentPolicyTask } from '../../tasks/isolate';
+import { login } from '../../tasks/login';
+import { ENDPOINT_VM_NAME } from '../../tasks/common';
+
+describe('Response console', () => {
+  const endpointHostname = Cypress.env(ENDPOINT_VM_NAME);
+
+  beforeEach(() => {
+    login();
+  });
+
+  describe('Isolate command', () => {
+    let response: IndexedFleetEndpointPolicyResponse;
+    let initialAgentData: Agent;
+
+    before(() => {
+      getAgentByHostName(endpointHostname).then((agentData) => {
+        initialAgentData = agentData;
+      });
+
+      getEndpointIntegrationVersion().then((version) => {
+        createAgentPolicyTask(version, (data) => {
+          response = data;
+        });
+      });
+    });
+
+    after(() => {
+      if (initialAgentData?.policy_id) {
+        reassignAgentPolicy(initialAgentData.id, initialAgentData.policy_id);
+      }
+      if (response) {
+        cy.task('deleteIndexedFleetEndpointPolicies', response);
+      }
+    });
+
+    it('should isolate host from response console', () => {
+      waitForEndpointListPageToBeLoaded(endpointHostname);
+      checkEndpointListForIsolatedHosts(false);
+      openResponseConsoleFromEndpointList();
+      performCommandInputChecks('isolate');
+      submitCommand();
+      waitForCommandToBeExecuted();
+    });
```

Comment:
In the second `test(){}` we expect endpoint to be isolated for we are releasing it there. I will leave my pov on this topic on the other comment.

Reply:
I see, although I think logically that check should go here, because it is the success condition for this test case. Let's say due to a bug, the command is successfully executed, but the endpoint is not isolated: in this case `isolation` test passes and the `release` test will fail, giving false indication about what feature went wrong.

Reply:
I see your point and I agree, changed.